README.md

```diff
@@ -15,6 +15,19 @@ The Standard for Public Code is in a draft format.
 We are preparing it for a version 1.0 release.
 Currently, we are testing it on a small number of codebases.
 
+## Applying the Standard for Public Code on your codebase
+
+If you want to apply the Standard for Public Code to your codebase, just go ahead, it's an open standard and free for anyone to use.
+To see how ready your codebase is, you can do a quick [eligibility self assessment](https://publiccodenet.github.io/assessment-eligibility) that will give you some rough expectations on how much work you may need to do to meet all criteria.
+
+The standard *should* be mostly self-explanatory in how to apply it on your codebase.
+If anything in the standard is unclear, we encourage you to open an issue here so that we can help you and everyone else who feel the same as you.
+For inspiration, look at the [Community built implementation guide](https://publiccodenet.github.io/community-implementation-guide-standard/) which contains examples and other tips.
```

Comment:
The hyperlink text doesn't match the title, hence it's odd to capitalize 'Community'. Instead, the whole hyperlink text can just be a description.
```suggestion
For inspiration, look at the [community built implementation guide](https://publiccodenet.github.io/community-implementation-guide-standard/) which contains examples and other tips.
```